backend/src/models/account/account.model.ts

```diff
@@ -29,7 +29,7 @@ const AccountSchema = new Schema<
 );
 
 AccountSchema.pre("save", function (next) {
-  var user = this;
```

Comment:
loz, nay thi dung var, hen la chu sua r chu neu k la bi chui sap mat

Reply:
you bích